SQL/create tables.sql

```diff
@@ -0,0 +1,51 @@
+
+
+
+
+
+CREATE TABLE Customers   
+(Customer_ID INT IDENTITY Not NULL UNIQUE,    
+CName NVARCHAR(16) NOT NULL,       
```

Comment:
We probably want to make the storage of CName larger than 16 to avoid any potential issues.